src/main/java/com/minecolonies/coremod/commands/GetColonyAndCitizen.java

```diff
@@ -0,0 +1,224 @@
+package com.minecolonies.coremod.commands;
+
+import com.minecolonies.coremod.colony.ColonyManager;
+import net.minecraft.command.ICommandSender;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+/**
+ * Get the Colony ID and Citizen ID out of a command.
+ */
+
+class GetColonyAndCitizen
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this class "final" or add a public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2974)